docs/scenarios/library-authoring/trimming.md

```diff
@@ -0,0 +1,141 @@
+# How to trim your package dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about trimming your package dependencies to use only the packages you need for .NET Core RC2.
+
+## What trimming means
+
+.NET Core RC2 introduces the .NET Standard Library metapackage, which is a NuGet package composed of other packages.  It is curated to be appropriate for developing libraries.  However, there's a good chance that your library won't use every single package it contains.  When authoring a library and distributing it over NuGet, it's a best practice to "trim" your dependencies down to only the packages you actually use.  This results in a smaller overall footprint for NuGet packages.
+
+## How to trim
+
+Currently, there is no official tooling which supports package trimming.  You'll have to do it manually.  The general process looks like the following:
+
+1. Reference `NETStandard.Library` version `1.5.0-rc2-24027` in a `dependencies` section of your `project.json`.
+2. Restore packages with `dotnet restore` from the command line.
+3. Inspect the `project.lock.json` file and find the `NETSTandard.Library` section.  It's near the beginning of the file.
+4. Copy all of the listed packages under `dependencies`.
+5. Remove the `.NETStandard.Library` reference and replace it with the copied packages.
+6. Remove references to packages you don't need.
+
+You can find out which packages you don't need by one of the following ways:
+
+1. Trial and error.  This involves removing a package, restoring, compiling, and seeing if your library still compiles.
```

Comment:
"compiling, and seeing if your library still compiles."
Maybe the first compiles can be removed.